app/content/analytics-toolbox-snowflake/overview/getting-access.md

```diff
@@ -38,15 +12,20 @@ The CARTO Analytics Toolbox contains two packages:
 This guide explains how to install the core package. In order to access the **advanced** features, please contact support@carto.com.
 {{%/ bannerNote %}}
 
-We can divide the process into two steps: setup and installation. The first one must be done only the first time, then the second one must be done every time you want to install a new version of the packages.
+The process consists of two steps: [setup](#setup) and [installation](#installation). The first one is required only the first time you install the toolbox, while the second one must be done every time you want to install a new version of the packages.
```

Comment:
the second one must be done every time you want to install a new version of the packages
->
the second one must be done every time you want to upgrade to a new version of the packages
?